test/specs/Conformance-test.js

```diff
@@ -0,0 +1,102 @@
+import _ from 'lodash';
+import React from 'react/addons';
+import stardust from 'stardust';
+import faker from 'faker';
+
+const TestUtils = React.addons.TestUtils;
+
+let getSDClassName = componentName => `sd-${_.kebabCase(componentName)}`;
+
+/**
+ * This test ensures all Stardust components conform to our guidelines.
+ */
+describe('Conformance', () => {
+  _.each(stardust, (SDComponent, name) => {
```

Comment:
Here we loop over all components in the Stardust object and run our conformance tests against them.  Since we have both the component name (Stardust object keys) and the actual component (Stardust object values), we can generate the test names, class names, test the class, and render the class.

This means we can easily test many aspects of the component and make test messages very readable.